apps/web/src/components/layout/components/HeaderNav.tsx

```diff
@@ -60,6 +73,15 @@ export function HeaderNav({ isIntercomOpened }: Props) {
   const segment = useSegment();
   const isSelfHosted = IS_DOCKER_HOSTED;
 
+  // Determine if the gizmo is visible
+  const [toggled, setToggled] = useState();
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🤓 nitpick:** ‏
```suggestion
  const [toggled, setToggled] = useState<boolean>(false);
```